### PR TITLE
09 place order front end

### PR DESCRIPTION
**Pull request by 1liu**
order

### CODE REVIEW

File: client/components/app.jsx [hunk 1]
```diff
@@ -58,6 +61,28 @@ export default class App extends React.Component {
...
+      .then(data => {
+        this.setState({ cart: [] });
+        this.setView('catalog', {});
+      })
+      .catch(error => console.log('Fetch cart failed!', error));
+  }
+
+  calcTotal() {
+    let total = 0;
+    this.state.cart.map(item => { total += item.price; });
```
Comment (thebearingedge) [inline client/components/app.jsx]: Avoid using `map` if you are ignoring the new array returned. Instead, use `forEach`

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: client/components/app.jsx (hunk 1 above)
Comment (1liu) [reply client/components/app.jsx]: fixed

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Please try to make the products grid appear not too far to the left, like the example.

**Pull request merged**